FAQ.md

```diff
@@ -85,6 +86,24 @@ Some image `.png` files are grayscale. This indicates that even though these ima
 
 It really depends on what image you're trying to change the colors of, where these colors are specified. Try looking for related files or `.pal` files.
 
+## How do I find absolute ROM offsets?
+When building external tools, it is usually necessary to know where the data you need is stored. 
```

Comment:
Have you been building tools that directly modify the ROM? You may want to be modifying the source files instead; this is what [Polished Map](https://github.com/Rangi42/polished-map), [Crystal Tracker](https://github.com/dannye/crystal-tracker), and [Tilemap Studio](https://github.com/Rangi42/tilemap-studio) do (as well as various Python scripts in the tools folder).